src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.ts

```diff
@@ -235,13 +239,18 @@ export class IscsiTargetDetailsComponent implements OnChanges, OnInit {
         });
       } else if (e.node.id.toString().startsWith('disk_')) {
         this.columns[2].isHidden = false;
-        this.data = _.map(this.settings.disk_default_controls, (value, key) => {
+        this.data = _.map(this.settings.disk_default_controls[tempData.backstore], (value, key) => {
           return {
             displayName: key,
             default: value,
-            current: tempData[key] || value
+            current: tempData.controls[key] || value
```

Comment:
```suggestion
            current: !_.isUndefined(tempData.controls[key]) ? tempData.controls[key]: value
```
@tspmelo  Otherwise it will not work with `0` values